localstack-core/localstack/services/events/provider.py

```diff
@@ -754,6 +760,26 @@ def _event_bus_dict_to_api_type_event_bus(self, event_bus: EventBus) -> ApiTypeE
 
         return event_bus_api_type
 
+    def _event_to_error_type_event(self, entry: PutEventsRequestEntry) -> str:
+        detail = (
+            json.dumps(json.loads(entry["Detail"]), separators=(", ", ": "))
```

Comment:
nit: assigning the loaded json to a variable could improve readability and makes debugging easier (e.g., if parsing fails)

Reply:
I would keep it as a list comprehension since this just adds a space to the separators if detail exists since aws has a strange response for errors in this particular case